src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -58,58 +66,197 @@
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          />
 
-        <v-col
-          cols="12"
-          sm="7"
-        >
-          ** Identifying number input goes here **
-        </v-col>
+          <v-col
+            cols="12"
+            sm="7"
+          >
+            <v-text-field
+              id="business-name"
+              v-model="business.identifier"
+              filled
+              label="Identifying Number"
+              :rules="getShowErrors ? identifyingNumberRules : []"
+            />
+          </v-col>
 
-        <v-col
-          cols="12"
-          sm="2"
-        >
-          <!-- empty column to line up with Undo button above -->
-        </v-col>
-      </v-row>
+          <v-col
+            cols="12"
+            sm="2"
+          >
+            <!-- empty column to line up with Undo button above -->
+          </v-col>
+        </v-row>
+
+        <!-- Business Name -->
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <label>Business Name in Home Jurisdiction</label>
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="7"
+          >
+            <v-text-field
+              id="business-name"
+              v-model="business.legalName"
+              filled
+              label="Business Name in Home Jurisdiction"
+              :rules="getShowErrors ? businessNameRules : []"
+            />
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="2"
+          >
+            <!-- empty column to line up with Undo button above -->
+          </v-col>
+        </v-row>
+
+        <!-- Business Number-->
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          />
+
+          <v-col
+            cols="12"
+            sm="7"
+          >
+            <v-text-field
+              id="business-number"
+              v-model="business.taxId"
+              filled
+              label="Business Number (Optional)"
+            />
+          </v-col>
+
+          <v-col
+            cols="12"
+            sm="2"
+          >
+            <!-- empty column to line up with Undo button above -->
+          </v-col>
+        </v-row>
+
+        <!-- Incorporation Date -->
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          />
+
+          <v-col
+            cols="12"
+            sm="7"
+          >
+            <DatePickerShared
+              id="incorporation-date"
+              ref="incorporationDateRef"
+              title="Incorporation Date in Home Jurisdiction"
+              :initialValue="getExistingBusinessInfo.incorporationDate"
+              :inputRules="getShowErrors ? incorporationDateRules : []"
+              :maxDate="getCurrentDate"
+              @emitDateSync="business.incorporationDate = $event"
+            />
```

Comment:
Check if you need `:nudgeRight="40"` and `:nudgeTop="85"` so the date picker displays correctly over the text input.

Compare with this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/a1491ac2-ddff-4cf4-9b54-f79e57fedf53)

Reply:
done